src/libraries/Errors.sol

```diff
@@ -21,6 +23,12 @@ library Errors {
                                  SABLIER-V2-LOCKUP
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @notice Thrown when trying to allow for hooks a contract that doesn't implement the interface correctly.
+    error SablierV2Lockup_AllowToHookIncorrectImplementation(ISablierRecipient recipient);
+
+    /// @notice Thrown when trying to allow for hooks an address with no code.
+    error SablierV2Lockup_AllowToHookZeroCodeSize(ISablierRecipient recipient);
```

Comment:
Why not `SablierV2Lockup_AllowToHookNotContract` ?

Reply:
Because it is [impossible to know if an address with no code is an EOA OR a contract being constructed OR a contract that got self-destructed](https://ethereum.stackexchange.com/q/15641/24693).

'zero code size' is a more generic phrasing than 'not contract'.